monogen/cmd/main.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+
+	"github.com/polymerdao/monomer/monogen"
+	"github.com/spf13/cobra"
+)
+
+var (
+	rootCmd = &cobra.Command{
+		Use:   "monogen",
+		Short: "monogen scaffolds a Monomer project.",
+		Long: "monogen scaffolds a Monomer project. " +
+			"The resulting project is compatible with the ignite tool (https://github.com/ignite/cli).",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			return monogen.Generate(cmd.Context(), appDirPath, goModulePath, addressPrefix, skipGit)
+		},
+	}
+
+	skipGit       bool
+	appDirPath    string
+	goModulePath  string
+	addressPrefix string
+)
+
+func main() {
+	ctx, cancel := signal.NotifyContext(context.Background(), syscall.SIGTERM)
```

Comment:
Should `syscall.SIGINT` also be included in the `NotifyContext` params in case the user interrupts the tool with Ctrl-C?

Reply:
Yes, thanks.